integration_test/third_party_apps_data/applications/couchbase/metadata.yaml

```diff
@@ -48,48 +45,62 @@ expected_logs:
         description: The name of the component that is issuing the cross datacenter log
 expected_metrics:
   - type: workload.googleapis.com/couchbase.bucket.operation.count
-    value_type: DOUBLE
+    value_type: INT64
     kind: CUMULATIVE
     monitored_resource: gce_instance
-    labels: {}
+    labels:
+      bucket_name: .*
+      op: .*
   - type: workload.googleapis.com/couchbase.bucket.item.count
-    value_type: CUMULATIVE
-    kind: DOUBLE
+    value_type: INT64
```

Comment:
`value_type` should be `INT64/DOUBLE` instead of `CUMULATIVE/GAUGE`; `kind` should be `CUMULATIVE/GAUGE` instead of `INT64/DOUBLE`